src/App/AppContainer.js

```diff
@@ -93,16 +91,17 @@ class AppContainer extends Component {
     }
 
     render() {
-		let { isFetchingResource } = this.props;
-
-		if (!this.state.isValidRoute) {
```

Comment:
now the 404 doesnt pop up you are on the correct path but wrong route, i think it's because you remove this line of code.